mne/io/eeglab/eeglab.py

```diff
@@ -445,3 +445,46 @@ def __init__(self, input_fname, events=None, event_id=None, tmin=0,
             reject=reject, flat=flat, reject_tmin=reject_tmin,
             reject_tmax=reject_tmax, add_eeg_ref=False, verbose=verbose)
         logger.info('Ready.')
+
+
+def _create_events_from_eeglab_raw(fname, event_id=dict()):
```

Comment:
Yes, I'd try putting it into the reader. I don't think we need a dedicated function. I'll probably refocus the reader to return an events channel directly.